test/abstract_client.js

```diff
@@ -7,6 +7,7 @@ var should = require('should')
 var sinon = require('sinon')
 var mqtt = require('../')
 var xtend = require('xtend')
+var assert = require('assert')
```

Comment:
we use should in this module.

Reply:
Noted. I will refactor all references that already uses "assert" to use "should".

Reply:
This is done.